data_safe_haven/utility/logger.py

```diff
@@ -67,7 +65,7 @@ class LogLevelHighlighter(RegexHighlighter):
     """
 
     base_style = "logging.level."
-    highlights = [
+    highlights: ClassVar[list[str]] = [
```

Comment:
This should be a tuple instead of a list. That should avoid the need for `ClassVar` here.